Sources/AuthFoundation/Network/APIRequest.swift

```diff
@@ -148,7 +148,22 @@ public protocol APIParsingContext {
 }
 
 extension APIParsingContext {
+    @_documentation(visibility: private)
+    public var codingUserInfo: [CodingUserInfoKey: Any]? {
```

Comment:
`UserInfo` may be confused with OIDC userInfo

Reply:
UserInfo is actually a common property name used throughout Apple's frameworks.  In this case, this corresponds to the [JSONEncoder's userInfo property](https://developer.apple.com/documentation/foundation/jsonencoder/2895176-userinfo), which is used to pass additional information needed to properly encode/decode data.